include/ebpf_program_types.h

```diff
@@ -55,6 +57,7 @@ typedef struct _ebpf_program_data
 {
     ebpf_program_info_t* program_info;
     ebpf_helper_function_addresses_t* helper_function_addresses;
+    ebpf_helper_function_addresses_t* global_helper_function_addresses;
```

Comment:
Since global helper functions are helper functions, the names in 59 and 60 are confusing since it implies that line 60 would be a subset of line 59.

Reply:
Updated.